app/views/insured/plan_shoppings/show.html.slim

```diff
@@ -348,7 +348,7 @@ javascript:
         networkTypes.forEach(function (networkType) {
           dataArray.forEach(function (enrollment) {
             if (networkType.toLowerCase() == 'dc-metro' || networkType.toLowerCase() == 'maine') {
-              if (enrollment.product.in_state_network) {
+              if (enrollment.product.dc_in_network) {
```

Comment:
can we avoid referencing specific states?

Reply:
dc_in_network is a field under components/benefit_markets/app/models/benefit_markets/products/product.rb.
& we are using that particular field under plan shopping page.

NOTE: It is being used  in many different places around the application, May be we have to create a new task to refactor state based references.